sdk/inc/azure/core/az_context.h

```diff
@@ -33,6 +33,7 @@ typedef struct az_context az_context;
  *
  * @details The root node in the tree (ultimate parent).
  */
+
```

Comment:
Revert this new line additions introduced in this PR between the existing docs and the struct definitions.